model_signing/signature/sigstore.py

```diff
@@ -1,106 +0,0 @@
-# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-"""This package provides the functionality to sign and verify models
-with sigstore."""
-from typing import Optional
-
-from absl import logging as log
```

Comment:
was deleting this file intentional?

Reply:
Yes. There's `signing/sigstore.py` that signs using the API we're building. This one was flagged when I changed `absl.logging` to just Python `logging` (to minimize dependencies): CodeQL reported that we were logging private details from the sigstore certificate. So, since the functionality was duplicated, I just deleted the file